pyrokinetics/normalisation.py

```diff
@@ -448,6 +466,41 @@ def set_lref(
             lambda ureg, x: x.to(ureg.lref_minor_radius).m * self.pyrokinetics.lref,
         )
 
+    def set_rhoref(
+        self,
+        local_geometry: Optional[LocalGeometry] = None,
```

Comment:
Could instead just pass `bunit_over_b0` in? Or maybe in a separate PR, completely remove the dependence of this file on `LocalGeometry` and `Kinetics` ? I think that was a mistake on my part